packages/GA4Client/test/src/tests.js

```diff
@@ -200,6 +200,38 @@ describe('Google Analytics 4 Event', function () {
 
             done();
         });
+
+        it('should set the measurement ID as an Integration Attribute', function (done) {
+            var sandbox = sinon.createSandbox();
+            var mPStub = sinon.stub(window.mParticle);
+
+            // Kit init requires checking SDK version, which in turn requires
+            // the SDK to be initialized. The actual version number is not
+            // relevant to capturing measurment ID.
+            mPStub.getVersion.returns('X.XX.X');
+
+            window.mockGA4EventForwarder = new mockGA4EventForwarder();
+
+            mParticle.forwarder.init(kitSettings, reportService.cb, true);
+
+            // GTAG will fire an async get request so we are simply
+            // verifying that measurementID/clientID makes it into
+            // mParticle.setIntegrationAttribute via gtag
+            dataLayer[2][3]('testMeasurementId');
```

Comment:
Should you be checking something here, as opposed to calling a function?  Since I couldn't test this in a browser, I couldn't tell what was happening here.   Do you mean that `dataLayer[2][3]` should equal `'testMeasurementId'`

Reply:
Maybe my comment wasn't clear. When running this through node, the mock gtag doesn't actually load the same way as in the browser, so I am manually triggering the gtag callback and verifying that `setIntegrationAttribute` is called with the proper arguments.